src/main/java/com/ghilly/service/CountryServiceRest.java

```diff
@@ -50,6 +52,7 @@ public Country getCountryById(int countryId) {
     @Override
     public void update(Country country) {
         exists(country.getId());
```

Comment:
the method can throw exception, but it is expected that boolean will return by name's method

Reply:
the method is renamed